gp-includes/template-links.php

```diff
@@ -29,7 +29,14 @@ function gp_link_get( $url, $text, $attrs = array() ) {
 	$attributes = gp_html_attributes( $attrs );
 	$attributes = $attributes ? " $attributes" : '';
 
-	return sprintf( '%1$s<a href="%2$s"%3$s>%4$s</a>%5$s', $before, esc_url( $url ), $attributes, $text, $after );
+	return sprintf(
+		'%1$s<a href="%2$s"%3$s>%4$s</a>%5$s',
+		$before,
+		esc_url( $url ),
+		$attributes,
+		' ' . $text,
```

Comment:
This doesn't seem right. The space character would also be linked. Looks like the space should be part of the `before` attribute or added to `$before` if it's not empty.

Reply:
I see, it does really make more sense to add in $before.
I'll wait for you to check the below screenshots before apply this suggestion.

Reply:
Solved on https://github.com/GlotPress/GlotPress-WP/pull/1210/commits/39ed1ac240e6fb69ecde4388a8f68772bdb39165